### ISSUE TITLE
tests installation

### ISSUE THREAD

**Issue by loriab** [MolSSI/QCEngine#92]
Somehow that manifest isn't sending the message that the `programs/tests/` need to be installed. (Hence psi's unhappiness, as its tests borrow data from qcng.)
```
python-scripts/qcengine
site-packages/qcengine-0.7.0.dist-info/INSTALLER
site-packages/qcengine-0.7.0.dist-info/LICENSE
site-packages/qcengine-0.7.0.dist-info/METADATA
site-packages/qcengine-0.7.0.dist-info/RECORD
site-packages/qcengine-0.7.0.dist-info/WHEEL
site-packages/qcengine-0.7.0.dist-info/entry_points.txt
site-packages/qcengine-0.7.0.dist-info/top_level.txt
site-packages/qcengine/__init__.py
site-packages/qcengine/_version.py
site-packages/qcengine/cli.py
site-packages/qcengine/compute.py
site-packages/qcengine/config.py
site-packages/qcengine/exceptions.py
site-packages/qcengine/extras.py
site-packages/qcengine/procedures/__init__.py
site-packages/qcengine/procedures/base.py
site-packages/qcengine/procedures/geometric.py
site-packages/qcengine/procedures/model.py
site-packages/qcengine/programs/__init__.py
site-packages/qcengine/programs/base.py
site-packages/qcengine/programs/cfour/__init__.py
site-packages/qcengine/programs/cfour/harvester.py
site-packages/qcengine/programs/cfour/keywords.py
site-packages/qcengine/programs/cfour/runner.py
site-packages/qcengine/programs/dftd3.py
site-packages/qcengine/programs/empirical_dispersion_resources.py
site-packages/qcengine/programs/entos.py
site-packages/qcengine/programs/gamess/__init__.py
site-packages/qcengine/programs/gamess/harvester.py
site-packages/qcengine/programs/gamess/runner.py
site-packages/qcengine/programs/model.py
site-packages/qcengine/programs/molpro.py
site-packages/qcengine/programs/mp2d.py
site-packages/qcengine/programs/nwchem.py
site-packages/qcengine/programs/psi4.py
site-packages/qcengine/programs/rdkit.py
site-packages/qcengine/programs/terachem.py
site-packages/qcengine/programs/torchani.py
site-packages/qcengine/programs/util/__init__.py
site-packages/qcengine/programs/util/hessparse.py
site-packages/qcengine/programs/util/pdict.py
site-packages/qcengine/stock_mols.py
site-packages/qcengine/testing.py
site-packages/qcengine/tests/__init__.py
site-packages/qcengine/tests/test_config.py
site-packages/qcengine/tests/test_procedures.py
site-packages/qcengine/tests/test_program_utils.py
site-packages/qcengine/tests/test_standard_suite.py
site-packages/qcengine/units.py
site-packages/qcengine/util.py
```

**Comment by loriab** [comment]
fwiw, qcel does look to install test files even in strange places. Maybe those engine ones just need a `__init__`? I haven't looked.

```
site-packages/qcelemental-0.4.2.dist-info/INSTALLER
site-packages/qcelemental-0.4.2.dist-info/LICENSE
site-packages/qcelemental-0.4.2.dist-info/METADATA
site-packages/qcelemental-0.4.2.dist-info/RECORD
site-packages/qcelemental-0.4.2.dist-info/WHEEL
site-packages/qcelemental-0.4.2.dist-info/top_level.txt
site-packages/qcelemental/__init__.py
site-packages/qcelemental/_version.py
site-packages/qcelemental/covalent_radii.py
site-packages/qcelemental/data/README.md
site-packages/qcelemental/data/__init__.py
site-packages/qcelemental/data/alvarez_2008_covalent_radii.py
site-packages/qcelemental/data/nist_2011_atomic_weights.py
site-packages/qcelemental/data/nist_2014_codata.py
site-packages/qcelemental/datum.py
site-packages/qcelemental/exceptions.py
site-packages/qcelemental/extras.py
site-packages/qcelemental/models/__init__.py
site-packages/qcelemental/models/align.py
site-packages/qcelemental/models/common_models.py
site-packages/qcelemental/models/molecule.py
site-packages/qcelemental/models/procedures.py
site-packages/qcelemental/models/results.py
site-packages/qcelemental/molparse/__init__.py
site-packages/qcelemental/molparse/chgmult.py
site-packages/qcelemental/molparse/from_arrays.py
site-packages/qcelemental/molparse/from_schema.py
site-packages/qcelemental/molparse/from_string.py
site-packages/qcelemental/molparse/nucleus.py
site-packages/qcelemental/molparse/pubchem.py
site-packages/qcelemental/molparse/regex.py
site-packages/qcelemental/molparse/to_schema.py
site-packages/qcelemental/molparse/to_string.py
site-packages/qcelemental/molutil/__init__.py
site-packages/qcelemental/molutil/align.py
site-packages/qcelemental/molutil/test_align.py
site-packages/qcelemental/periodic_table.py
site-packages/qcelemental/physical_constants/__init__.py
site-packages/qcelemental/physical_constants/context.py
site-packages/qcelemental/physical_constants/ureg.py
site-packages/qcelemental/testing.py
site-packages/qcelemental/tests/__init__.py
site-packages/qcelemental/tests/addons.py
site-packages/qcelemental/tests/test_constants.py
site-packages/qcelemental/tests/test_covalentradii.py
site-packages/qcelemental/tests/test_datum.py
site-packages/qcelemental/tests/test_importing.py
site-packages/qcelemental/tests/test_model_serials.py
site-packages/qcelemental/tests/test_molecule.py
site-packages/qcelemental/tests/test_molparse_align_chiral.py
site-packages/qcelemental/tests/test_molparse_from_schema.py
site-packages/qcelemental/tests/test_molparse_from_string.py
site-packages/qcelemental/tests/test_molparse_parse_nucleus_label.py
site-packages/qcelemental/tests/test_molparse_pubchem.py
site-packages/qcelemental/tests/test_molparse_reconcile_nucleus.py
site-packages/qcelemental/tests/test_molparse_to_schema.py
site-packages/qcelemental/tests/test_molparse_to_string.py
site-packages/qcelemental/tests/test_molparse_validate_and_fill_chgmult.py
site-packages/qcelemental/tests/test_periodictable.py
site-packages/qcelemental/tests/test_testing.py
site-packages/qcelemental/tests/test_units.py
site-packages/qcelemental/tests/test_utils.py
site-packages/qcelemental/util/__init__.py
site-packages/qcelemental/util/gph_uno_bipartite.py
site-packages/qcelemental/util/importing.py
site-packages/qcelemental/util/internal.py
site-packages/qcelemental/util/itertools.py
site-packages/qcelemental/util/misc.py
site-packages/qcelemental/util/np_blockwise.py
site-packages/qcelemental/util/np_rand3drot.py
site-packages/qcelemental/util/scipy_hungarian.py
site-packages/qcelemental/util/test_gph_uno_bipartite.py
site-packages/qcelemental/util/test_scipy_hungarian.py
```

**Comment by Lnaden** [comment]
I believe its because that `programs/tests` does not include a `__init__.py` file, hence the `setuptools.find_packages()` will not work according to its docs:

> Packages are only recognized if they include an __init__.py file (https://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages)

I can add that and then do a 0.7.1 (which I believe is the correct way to do patches)

**Comment by Lnaden** [comment]
Fixed by #93
